back-end/db/player.ts

```diff
@@ -104,21 +104,26 @@ export default {
   playerAnswerQuestion: async (playerId: string, guess: number, gameData: IGame): Promise<any> => {
     try {
       const player: IPlayer | null = await Player.findOne({id: playerId});
-
+      // Log player data
+      // console.log(playerId + " | " + guess);
+      // console.log(gameData);
+      
       if (player === null) {
         throw `Player not found: ${playerId}`;
       } else {    
         const newQuizGuesses = player.quizGuesses;
         newQuizGuesses[gameData.currentQuestionIndex] = guess;
-
+        // console.log(player.name + "'s score before guess: " + player.score);
```

Comment:
most of the comments like this one were to verify that the score was indeed being added under player data